android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -303,6 +306,16 @@ class AirshipPlugin : MethodCallHandler {
         result.success(true)
     }
 
+    private fun setDataCollectionEnabled(call: MethodCall, result: Result) {
+        UAirship.shared().isDataCollectionEnabled = call.arguments as Boolean
+        result.success(true)
+    }
+
+    private fun setPushTokenRegistrationEnabled(call: MethodCall, result: Result) {
+        UAirship.shared().pushManager.pushTokenRegistrationEnabled = (call.arguments as Boolean)
```

Comment:
Why is `call.arguments as Boolean` wrapped in parentheses here, but not a couple lines above?

Reply:
I converted these from their setters, because that's what kotlin likes, no strong reason